portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -30,7 +31,7 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that displays and update comments on the web page. */
```

Comment:
what do you mean by update? this seems to me like it just gets the comments. "update" sounds like it can update the DB.

Reply:
Should just be display.